app/docker-compose.yml

```diff
@@ -31,3 +31,14 @@ services:
       - postgres
     links:
       - postgres
+  media:
+    build: media
+    restart: always
+    env_file: media.dev.env
+    volumes:
+      - "./data/media/uploads/:/uploads/"
```

Comment:
Is this so we can actually delete the images?

Reply:
I just copied what aapeli told me